local_env_samples/multiple_model_training_sample.py

```diff
@@ -28,16 +29,16 @@ def read_by_line():
 
 # Save training dataset (streamed)
 dataset_response: SaveDatasetResponse = sdk.save_dataset(read_by_line(),
-                                                         "sample-dataset",
-                                                         overwrite=True)
+                                                         "sample-dataset")
```

Comment:
Out of curiosity why did we get rid of the overwrite parameter here?

Reply:
It was changed so that a dataset with the same name would be re-named as `same-dataset-name (1)`, `same-dataset-name (2)`, etc. so overwriting is no longer necessary